### ISSUE TITLE
Auto click error - Xcode (Code 2)

### ISSUE THREAD

**Issue by anushkmittal** [iam4x/pokemongo-webspoof#5]
execution error: System events got an error: osascript is not allowed assistive access . (-1719)

**Comment by stefan-o** [comment]
Goto System Preferences -> Security & Privacy -> Privacy Tab -> Accessibility in the List -> Then Authorize your self by clicking the lock in the lower left hand corner, then in the right list, check pokemongo-webspoof.

**Comment by anushkmittal** [comment]
Yes, it works! I was trying to find a to allow Apple Scripts like in earlier versions of macOS but yes, this works!

**Comment by sagestein461** [comment]
didn't fix it for me...

**Comment by g3crossfire** [comment]
Didn't fix it for me.

**Comment by iam4x** [comment]
@sagestein461 @g3crossfire 

1. Focus Xcode
2. Into menu bar, click Xcode, click into menu Debug -> Simulate Location -> pokemonLocation
3. Try to move on the map, if it stills fails click into menu Debug -> Simulate Location -> Don't Simulate Location

If it stills fails, do these 3 steps again and again it works for me after severals tries

I get this error sometime, can't find out why I supposed it was macOS Sierra beta. On what OS are you running?

**Comment by iam4x** [comment]
Added on the README https://github.com/iam4x/pokemongo-webspoof#known-issues

**Comment by anushkmittal** [comment]
I'm again getting the same error even after trying all workarounds

Error Auto Click Xcode - Code 2

/Content/Resources/app/autoclick.applescript execution error: System events got an error: osascript is not allowed assistive access . (-1719)

**Comment by iam4x** [comment]
@anushkmittal try to:

* Delete the app
* Remove `pokemongo-webpsoof.app` from authorized app to Accessibility
* Download last release
* Add new `pokemongo-webspoof.app` into authorized app to Accessibility

**Comment by anushkmittal** [comment]
@iam4x Yes! This works -:)  Unfortunately the Pokémon Go servers are down atm.

**Comment by anushkmittal** [comment]
@iam4x I was thinking that this project could be really way beyond GPS-Spoof if you think about it in other terms. This could be the perfect solution for all location based app developers who would like to simulate more complex GPS locations than which could be found by static or directional GPX files.

**Comment by iam4x** [comment]
Will think about it, you are totally right!

**Comment by Qabd88** [comment]
im trying to change simulate location under Debug but all options are faded

**Comment by jayantr94** [comment]
1.
The provisioning profile "iOS Team Provisioning Profile: com.iam4x.pokemon-webspoofjr94" is expiring soon
warning: skipping file '/Users/jayantrao/Desktop/pokemongo-webspoof-master/pokemonLocation.gpx' (unexpected file type 'text' in Frameworks & Libraries build phase)

2. I don't get the Location file on Debug - Simulate Location

3. Nor do I get - Check the `Auto update Xcode location` in the app when everything is running

4. When I am on Pokemon Go and Xcode opens on my phone, it's all white. :/

**Comment by sokki** [comment]
How to deal with `Accessibility` when running through `npm run dev` ?

**Comment by iam4x** [comment]
Thank's @sokki will add a note on the README.md

**Comment by voidstarfire** [comment]
Tried all of above. None are working.

Still getting
<img width="322" alt="screen shot 2016-07-19 at 9 22 30 pm" src="https://cloud.githubusercontent.com/assets/6940726/16963476/edea2da0-4df6-11e6-8820-b3144625e214.png">

**Comment by voidstarfire** [comment]
Scratch that, people don't use the quick route it breaks:

Breaks: 
<img width="533" alt="screen shot 2016-07-19 at 9 24 38 pm" src="https://cloud.githubusercontent.com/assets/6940726/16963541/381d0582-4df7-11e6-8da6-a588f51fd506.png">

Works:
<img width="584" alt="screen shot 2016-07-19 at 9 25 04 pm" src="https://cloud.githubusercontent.com/assets/6940726/16963554/46d26860-4df7-11e6-8d0b-e9a541efeb9c.png">

**Comment by sibor64** [comment]
Hello
I have the same problem, I have to enable or disable location in order to teleport.
Once on the place, I can not move I'm forced to activate or disable location

**Comment by voidstarfire** [comment]
Updated to 1.1.0; issue now exists again. Won't auto-update 👎

**Comment by Talhabestaz** [comment]
My problem is bigger, no message pop up but location still not change on iphone :S

**Comment by bentheone** [comment]
I can't use pokemonLocation.
The pokemonLocation.gpx is red in my project.
help?

**Comment by Donald181** [comment]
Hallo,
i have a Problem, my problem is when i start Debug-->Location--> Pokemonlocation i get this error in the webspoof app 
Can anyone help me ?

<img width="800" alt="error pokemon" src="https://cloud.githubusercontent.com/assets/20566675/17004349/db98669c-4ed4-11e6-8bfc-8d615e751b05.png">

My second Problem is wenn i typ an Destination i get that error see below


<img width="797" alt="error22" src="https://cloud.githubusercontent.com/assets/20566675/17004534/b6306160-4ed5-11e6-8afd-5ac5e5d700ed.png">

Please Help me 

Thank you

**Comment by dubert** [comment]
@Donald181 I have the same issue. It's having issues detecting current location.

**Comment by GNSPS** [comment]
+1 @Donald181 same here. But I successfully ran it once after the update... Can't figure out why this would behave like this now.

**Comment by tylercwats** [comment]
"An App ID with Identifier 'com.iam4x.pokemon-webspoof' is not available. Please enter a different string."

Was working for me earlier. Now I'm getting this error.

**Comment by GNSPS** [comment]
Guys I was just debugging things and got it working by entering the application's contents and changing the Info.plist to change its bundle identifier and the display name after deleting its accessibility rights.

So to sum up:


- Delete the app
- Remove pokemongo-webpsoof.app from authorized app to Accessibility
- Change Info.plist's Bundle Identifier and Display Name inside app's contents to something different
- Add new pokemongo-webspoof.app into authorized app to Accessibility


BONUS: If you're feeling like it while you're at it change also the xcode project bundle identifier so it persists between every opening of the app

**Comment by Vvkmnn** [comment]
Just tried the above fix. Unfortunately, didn't work for me.

**Comment by pony-boy** [comment]
Hi, I had this working great earlier (kudos by the way), but now when I start the app it doesn't find my current location it put me in the middle of the atlantic ocean. Is there something I can do to force it to recognize my location?

**Comment by lpre30** [comment]
I am having the same issue where I am somewhere in the atlantic

**Comment by akkarachaiwangcharoensap** [comment]
Hey, I think I found a solution to the "unknown location" problem. I am not sure what the hell happened but this solves the problem. First download the previous version (1.0.2), https://github.com/iam4x/pokemongo-webspoof/releases/download/v1.0.2/pokemongo-webspoof.app.tar.gz. Remove everything you have on the latest version 1.10. Build and launch the previous version. Then after that remove the previous version and download the latest release. Build and compile, I think that should work. Note, you need to remove the authorized the accessibility and start from scratch. I hope this helps.

**Comment by vvit** [comment]
Guys, I've sometimes simulating location doesn't work perfectly in Xcode. So after getting the `Error Auto Click Xcode` try to turn off and then turn on simulation of location.

**Comment by RubenHiem** [comment]
Does it work with iOS 10 and Xcode-beta?

**Comment by Vvkmnn** [comment]
@RubenHiem No, the same error occurred, but it worked for me last night for some reason. Covered it in #97

**Comment by RubenHiem** [comment]
@Vvkmnn so its the best to downgrade my phone back to 9.3?

**Comment by coding102** [comment]
Any clue ???

Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid.

**Comment by edgeui** [comment]
Looks like i cant collect pokemon using latest update, just walk

**Comment by coding102** [comment]
I got the solution! You need to give permission to the "Terminal.app" too.  Go to accessibility > Privacy and make sure "Terminal.app" is also checked along with the "Script Editor.app"

**Comment by pol0nium** [comment]
@Donald181 workaround : force random gps coordinates (not pointing in the sea) in the app interface, then search for your adress in the search bar and teleport to location.
It seems that the bug exists because google cannot compute directions from the atlantic.

**Comment by namsiuh** [comment]
@coding102 where do i do this?

**Comment by coding102** [comment]
system preferences > Security and Privacy > Privacy tab > make sure terminal.app is checked.

**Comment by namsiuh** [comment]
tHIS IS WHAT I'M GETTING PLEASE HELP

![screen shot 2016-07-21 at 10 49 37 pm](https://cloud.githubusercontent.com/assets/20589483/17045190/77246fc8-4f95-11e6-884d-b7d1daa9bd32.png)

**Comment by namsiuh** [comment]
@coding102 I dont have that as an option.
![screen shot 2016-07-21 at 10 51 44 pm](https://cloud.githubusercontent.com/assets/20589483/17045228/badad4aa-4f95-11e6-8dc7-1a55dc9438d6.png)

**Comment by coding102** [comment]
That is odd, it is there for me.
<img width="1440" alt="screen shot 2016-07-21 at 11 04 14 pm" src="https://cloud.githubusercontent.com/assets/14987467/17045481/80ebf51a-4f97-11e6-9222-9d5f1a174f0d.png">

**Comment by coding102** [comment]
Is your local server running?

**Comment by namsiuh** [comment]
@coding102 this obviously way above me. Thanks for tryin though

**Comment by HunterGeo** [comment]
Hi
It seems not work for me.
After all settings down, it can not update the location where I am.

**Comment by phreaky1234** [comment]
How do I solve "Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid? It says it failed to parse the file.

**Comment by markadrake** [comment]
When I go to the simulate location -> pokemon step I'm actually met with this issue:

![image](https://cloud.githubusercontent.com/assets/713355/17057652/66a90646-4fe9-11e6-98c7-9d4a849a669c.png)

**Comment by bigbangtheory22** [comment]
Hi,

I seem to be stuck with this error message:

/Users/davidong/Downloads/pokemongo-webspoof-darwin-x64/pokemongo-webspoof.app/Contents/Resources/app/autoclick.applescript:55:187: execution error: System Events got an error: Can’t get menu item "pokemonLocation" of menu 1 of menu item "Simulate Location" of menu 1 of menu bar item "Debug" of menu bar 1 of process "Xcode". (-1728)

I've added pokemongo-webspoof, Script Editor and Terminal to accessibility without luck.

Cheers!

**Comment by bigbangtheory22** [comment]
Thank you very much @g3crossfire ! I have just tried that then and it comes up with this error:

/Users/davidong/Downloads/pokemongo-webspoof-darwin-x64/pokemongo-webspoof.app/Contents/Resources/app/autoclick.applescript:55:187: execution error: System Events got an error: Can’t get menu bar 1 of process "Xcode". Invalid index. (-1719)

Thanks a million for your help!

**Comment by g3crossfire** [comment]
@bigbangtheory22 
step 1 : Rename the Bundle Indentifer to something unique and different (for example com.[yourgithubname].pokemon-webspoof - you need it after an update)

step two: delete all apps and permissions linking to it... then  give permission to the "pokemongo-webspoof.app" and "Terminal.app" . Go to accessibility > Privacy and make sure "Terminal.app" is also checked along with the "Script Editor.app"

**Comment by Seonx** [comment]
Hi,
It worked for me once, but now It does not recognize my initial (current) location when starting the app.
Im getting this error:
![schermafbeelding 2016-07-22 om 20 22 19](https://cloud.githubusercontent.com/assets/20602474/17067100/77e967d2-504a-11e6-865d-5a5bc8852bb2.png)

Hope someone can help!

**Comment by masuhara** [comment]
@markadrake Me, too.

Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid.
Failed to parse GPX file (/Users/.../pokemongo-webspoof-master/pokemonLocation.gpx)

**Comment by Seonx** [comment]
Okay but how can I fix that?

**Comment by woshahua** [comment]
I have the same issue too,  and my phone's gps get changed to the wrong location in the app, how to fix the gps location of iphone.

**Comment by jimmyleejr** [comment]
I am so lost I cant even open the .app to even start the xcode

**Comment by woshahua** [comment]
the location showed in  locationSpoof is wrong, how to fix it to the right place where i am?

**Comment by lvissxz** [comment]
Getting the error "Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid".

**Comment by bigbangtheory22** [comment]
Okies I'm sure someone has already done this but this is a step by step as to how I got it sorted in the end.

1. Download XCode
2. Download pokemongo-webspoof app
3. Make sure you register for a developer account with your Apple ID: https://idmsa.apple.com/IDMSWebAuth/login?appIdKey=891bd3417a7776362562d2197f89480a8547b108fd934911bcbea0110d07f757&baseURL=https://developer.apple.com/&path=%2Fregister%2Fagree%2F&rv=1
4. Start Pokémon Go on your iOS device
5. Connect to your Mac via USB
6. Rename the "Bundle Indentifier" to something unique and different (for example com.[yourgithubname].pokemon-webspoof - you need it after an update)
7. Click [Team] below "Version" and "Build" and select your Apple Developer account
8. Click on [Product] in the menu bar and select [Destination]
9. Under [Device} you will see your connected iOS device
10. Click [Product] in the menu bar again and then select [Build]
11. After the build is complete, click [Run] and then you may get a message which says, "Cannot launch "Pokemon Webspoof".
12. To fix this open your Mac computer and then go to System Preferences -> Security & Privacy -> Privacy Tab -> Accessibility and make sure that the "pokemongo-webspoof" is ticked.
13. Select [+] and then go to your "Utilities" folder and add "Terminal" and "Script Editor" to the apps which are allowed to control your computer
14. Click [Product] in the menu bar again and then select [Build]
15. After the build is complete, click [Run] and the app should appear on your iOS device
16. It'll then give you the option "Launch Pokemon Go" you should proceed with the instruction
17. Select the "Auto update Xcode location" in top corner of the Pokemon Go Webspoof app when everything is running
18. Go back to Xcode, click into menu [Debug], select [Simulate Location], then select [pokemonLocation]
19. Attempt to move your character with the arrows on the Pokemon Go Webspoof app
20. If this doesn't work, go back to Xcode, click into menu [Debug], select [Simulate Location], then choose [Don't simulate location]
21. Go back to the Pokemon Go Webspoof app and attempt to move your trainer and it should work now. I talked a mate through it that had never done this before, so it should work but if it doesn't I have no idea!

I'm sure I may have added steps which were unnecessary but this was my first go at playing with this sort of stuff and is what worked for me.

A note of caution, just make sure that when you open the Pokemon Go Webspoof app when it's all up and running, that you are where the map says you are or you may cop a soft ban.

Also don't move too quickly between places or they may pick up on it and soft ban you too.

Cheers and all the best!

**Comment by jopower** [comment]
Okay, completely new to all these and have no idea about using Xcode or anything like it. 

Just wondering if someone can break it down to me lol. 

Firstly, what is the Bundle Identifier and where about do i find it? it that just the name of the downloaded file or an actual file inside of x code? 

Secondly, any one else having trouble trying to sign up on the developers page? i keep getting this error message 'The developer.apple.com page isn’t working" 

Probably have a lot more question but i'm not that far in ha!

**Comment by bigbangtheory22** [comment]
@jopower check the updated post above yours for the extra step but if you can't access the developers page you may have trouble with the rest.

**Comment by mmk1011** [comment]
Hey guys. I got the xcode file to run and change my location to New York but I can't figure out how to launch the app that lets me navigate through Time Square. I'm on a Mac using the XCode app from the Apple store.

**Comment by mmk1011** [comment]
Just launched the app. When i try to catch a pokemon, it runs as soon as I catch it with a Pokeball. Pokestops don't seem to work. I keep spinning the circle and nothing happens. Does anyone know how to fix this? Thanks!!

**Comment by namsiuh** [comment]
@mmk1011 when this happens it means you have been soft banned. It temporarily disable the ability to cath any thing including items fromnpokestops for abising the gps of the game. The good news is that its only for a couple hours.The only way this doesnt happen is by stayin around the area you live in. As long as you travel somewhere by car or walking it shouldnt happen. I live close to a park so luckily i dont have to go far, but u wanna stay away from teleport to avoid this from happening.

**Comment by Frozenpopola** [comment]
if i try to teleport to other countries, it return a ZERO_RESULTS Error.. someone experienced this?
i'm from Brazil and if i teleport inside Brazil it works, but other countries don't

**Comment by namsiuh** [comment]
@frozenpopola you cant teleport over oceans. It wont work. Only over land.

**Comment by anushkmittal** [comment]
Work around, change the coordinates manually by checking it through Google Maps. Then you can navigate around that area easily

**Comment by Frozenpopola** [comment]
@anushkmittal i've tried to do that but it return an error NaN coordinates


@namsiuh it makes sense thanks

**Comment by jonoboiiy** [comment]
@bigbangtheory22  that was extremely clear, thank you for that. After following those instructions I don't think anyone will have any issues..... until the next issue :)

**Comment by anushkmittal** [comment]
@Frozenpopola you have to be careful and edit the coordinates rather than using 'delete' and rewriting it which won't be possible..

**Comment by ilias412** [comment]
@bigbangtheory22 @iam4x hey, im new here, and i have, i guess, 2 problems.

 first of all the pokemon web spoof appears on my make with an crossed out emblem. what does that mean? 
<img width="462" alt="bildschirmfoto 2016-07-27 um 16 42 13" src="https://cloud.githubusercontent.com/assets/20684951/17179436/84d17502-5419-11e6-966e-c26a52fce35c.png">
the second weird thing is, that i get this message when i wnat to launch the pokemon-location: _"Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid. Failed to parse GPX file (/Users/Daniel/Desktop/pokemongo-webspoof-master/pokemonLocation.gpx)"_ 

what shall i do? please excuse my bad english. thx

**Comment by Mr10001** [comment]
Hi, this thread is great. 

However, I am unable to locate the pokemongo-webspoof.app file to add to my privacy section to allow to control my computer. Also, when I run the app, it comes up on my iOS device, I click launch pokemon go and then nothing else happens?

Any tips?

**Comment by Mr10001** [comment]
I actually just found the .app file after looking in the project within xcode. It was under: pokemon-webspoof > Products > Pokemon Webspoof.app

The directory was listed on the right side as being under:

Users/YOURUSERNAME/Library/Developer/Xcode/DerivedData/pokemon-webspoof-efvpbovblhrvsydtlnvxnoqhllwz/Build/Products/Debug-iphoneos/Pokemon WebSpoof.app

I simply navigated to this directory and then dragged and dropped the .app file to the privacy section.

**Comment by Mr10001** [comment]
I'm stuck on step 18. "Select the "Auto update Xcode location" in top corner of the Pokemon Go Webspoof app when everything is running"

I do not see this on my iOS device within the webspoof app, all I see is the option to "Launch Pokemon Go Now", which I do, it brings me back to the Pokemon Go app.

I do not see anything on my computer?

Any thoughts?

**Comment by EnglandBaby** [comment]
Guys help me i have a problem: 1st -XCODE dosen't support phone's IOS 9. 2nd- I cant rename bundle identifier

**Comment by Yomerito** [comment]
Is there a way to run this in iOS 10?, I've been trying but keep getting the could find developer disk image message, anyone has any ideas or do I have to forcibly get back to iOS 9?

**Comment by nickg819** [comment]
You need xcode 8 beta for ios 10 support.

**Comment by xXFettiXx** [comment]
Hello , I do not understand the step to install a developer account !
can you help me? everything else works but I'm disconnected every 5 minutes and this forces me to repeat this process indefinitely
thank you in advance

**Comment by nlenus** [comment]
I tried to install the app via terminal commands and I get this loads of warnings- is this normal? When I run it, xcode opens and also an app called electron but it's a blank white page.

npm WARN package.json pokemongo-webspoof@1.2.0 No repository field.
npm WARN package.json pokemongo-webspoof@1.2.0 No bin file found at ./bin/pgws
npm WARN package.json pokemongo-webspoof@1.2.0 No bin file found at ./bin/pgws
npm WARN package.json Dependency 'electron-prebuilt' exists in both dependencies and devDependencies, using 'electron-prebuilt@^1.3.0' from dependencies
npm WARN package.json Dependency 'google-map-react' exists in both dependencies and devDependencies, using 'google-map-react@^0.16.3' from dependencies
npm WARN deprecated electron@0.4.1: The original electron project has been moved. Visit github.com/logicalparadox/electron for more details.
npm WARN peerDependencies The peer dependency babel-core@^6.0.0 included from babel-loader will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency eslint@^2.9.0 included from eslint-config-airbnb will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency eslint-plugin-jsx-a11y@^1.2.0 included from eslint-config-airbnb will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency eslint@^2.9.0 included from eslint-config-airbnb-base will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency angular@< 2 included from autocomplete.js will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency jquery@>= 1.7 included from autocomplete.js will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.

**Comment by anguyen56** [comment]
im getting Location simulation cannot continue because the GPX file "pokemonLocation.gpx" is invalid. please help

**Comment by cfras104** [comment]
I get error: WebSpoof[1033:56683] -canOpenURL: failed for URL: "b335b2fc-69dc-472c-9e88-e6c97f84091c-3://" - error: "(null)"

Could this be because of the recent update? Or is it me.

**Comment by patrickjeu** [comment]
Same as above ^. I get error: WebSpoof[1033:56683] -canOpenURL: failed for URL: "b335b2fc-69dc-472c-9e88-e6c97f84091c-3://" - error: "(null)"

This error occurs when pressing the Launch Pokemon Go button.

Could this be because of the recent update?

**Comment by Mr10001** [comment]
Check out the following for the canopenurl failed error:

https://github.com/iam4x/pokemongo-webspoof/issues/302

**Comment by R333166** [comment]
Hi to all,
This is what is happening to me (simulate location is not enabled). Any help?
<img width="701" alt="schermata 2016-08-02 alle 11 37 12" src="https://cloud.githubusercontent.com/assets/20791732/17324249/df1139ae-58a5-11e6-92bc-1173bf090a37.png">

**Comment by elvquant** [comment]
locking around my issue there is no menu entry "pokemonLocation" and I haven't found any "*.gpx" which are able to select in Xcode Debug menu ... I found: http://stackoverflow.com/questions/16780366/xcode-greys-out-gpx-files-in-filebox

May be the problem is for both issues (missing entry and unable to load gpx) are possible, that there is an misspelling contenttype-mapping on the system. 

I have no solution at the moment ... ;-(

**Comment by bullseye1975** [comment]
I added all the code and it compiled just fine on my phone. But the map/controller doesn't launch on my Mac.  I went to Accessibility and I can't add it.  Any other suggestions?

**Comment by elvquant** [comment]
After installing a fresh minimal OS X 10.10 and Xcode 7.3.1 all works correctly!
The contenttype-mapping is now correct and the correct icon will shown.

Now I have to check, what I can do for my "regular" working system ...

**Comment by boom-time** [comment]
what actually updates the pokemonLocation.gps file because I have everything working except that file always remains empty so xcode complains if I have the auto update location on  (running in dev mode)

**Comment by cesarvh** [comment]
what is the exact format of the pokemonLocation.gpx?
lat = "xxx"
lon = "yyy"??
"xxx, yyy" ??

halp

**Comment by elvquant** [comment]
https://www.google.de/#q=gpx+file+format

**Comment by grycyk** [comment]
running fine here on Xcode 8 and iOS 10, just can't seem to get PokeStops for give me anything...

**Comment by Yomerito** [comment]
Anyone having trouble with the latest iOS 10 beta update (14A5341a) ? I just installed it and I keep getting the "unknown error" message or "couldn't read from device", been looking for a while but I can't find the problem. Any ideas or suggestions?

**Comment by MoOx** [comment]
@Yomerito getting the same issue since the latest update. I tried things here http://stackoverflow.com/questions/38963815/xcode8-0-beta-6-unable-to-read-from-device but nothing worked for me. I can't use this project for now :(

**Comment by Yomerito** [comment]
I'm on the same boat, tried everything listed there and some other ideas but nothing works :/ worst is I can't use any project and basically Xcode became useless since I don't have another iPhone, hope they fix it soon, ill keep trying and post it if I can solve the problem. By the way did you try returning the iPhone to factory values, for some reason I can't do it, not sure if it is part of the same problem.

**Comment by asonephady** [comment]
@iam4x i still get code 2 error every time i open up the app. it usually takes like 5 minute (even if i toggle debug sim location on and off) for it to start working again lol. have to sit through 5 minutes of the error then it'll let me walk. aynthing i can do?

**Comment by bgou** [comment]
can someone try this?

https://jacobsalmela.com/2014/06/12/bash-script-enable-access-assistive-devices-programmatically-os-x-mavericks-10-9-x-simulate-keystrokes/

**Comment by chadnielsen** [comment]
Running High Sierra 10.13.4 and iOS 11.3. I was able to solve this by adjusting the Deployment Target to 11.3. I saved the project, quit everything, then reopening everything again per the Read Me. Works great!

**Comment by Yuri224488** [comment]
I CAN'T UPDATE MY COMPUTER  TO USE THIS SOMEONE PLEASE MAKE THIS WORK ON OSX 10.8.5

**Comment by kyle7908** [comment]
Got It working perfect, trouble I am having is getting it to go to Santa Monica pier, it does uk and Europe brilliant just not USA. It will teleport me straight back. Any ideas? Thanks

**Comment by WebSpoofer** [comment]
You need to enter a lat and log in the US and then you can move around in the US by entering addresses. Good Luck.
